src/components/Avatar/Avatar.component.tsx

```diff
@@ -262,14 +277,27 @@ const Avatar: FC<AvatarProps> = ({
       {background?.src && <Box {...background} />}
       {capture && <Capture {...capture} />}
       {style?.background && <BackgroundColor color={style.background as string} />}
-      <EffectComposer disableNormalPass>
+      <EffectComposer multisampling={0} autoClear={false}>
         <Bloom
-          luminanceThreshold={bloom?.luminanceThreshold}
-          luminanceSmoothing={bloom?.luminanceSmoothing}
-          intensity={bloom?.intensity}
-          kernelSize={bloom?.kernelSize}
-          mipmapBlur={bloom?.mipmapBlur}
+            luminanceThreshold={bloom?.luminanceThreshold}
+            luminanceSmoothing={bloom?.luminanceSmoothing}
+            intensity={bloom?.intensity}
+            kernelSize={bloom?.kernelSize}
+            mipmapBlur={bloom?.mipmapBlur}
```

Comment:
What's causing this auto-reformatting?

Reply:
reformat code, for some reason, we miss the .editorconfig where we set those standards